### PR TITLE
Add recipe for loophp/psr17 package.

### PR DESCRIPTION
**Pull request by drupol**
| Q             | A
| ------------- | ---
| License       | MIT

Link: https://github.com/loophp/psr17

### CODE REVIEW

File: loophp/psr17/1.0/config/packages/loophp_psr17.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+services:
+    # Register loophp/psr17/Psr17 class and autowire/autoconfigure it.
+    loophp.psr17.psr17:
```
Comment (Nyholm) [inline loophp/psr17/1.0/config/packages/loophp_psr17.yaml:3]: Is there a reason you don’t use the class name as id?

Comment (drupol) [reply loophp/psr17/1.0/config/packages/loophp_psr17.yaml:3]: True, I will update!

Comment (drupol) [reply loophp/psr17/1.0/config/packages/loophp_psr17.yaml]: Updated!

**Review by Nyholm — APPROVED** [review]
Thank you

**Comment by drupol** [comment]
Thanks !!!

**Comment by Nyholm** [comment]
@symfony-flex-server merge now

**Review by Nyholm — APPROVED** [review]
Merge now

**Comment by drupol** [comment]
sudo merge now ? :laughing:

**Comment by Nyholm** [comment]
It seams like I don’t have the power to merge it.

**Comment by drupol** [comment]
Ok...

**Comment by Nyholm** [comment]
Wohoo. Thank you

**Comment by drupol** [comment]
Merci :)

**Pull request merged**